src/wp-includes/functions.php

```diff
@@ -8478,7 +8478,14 @@ function is_virtualbox() {
 		return $is_virtualbox;
 	}
 
-	// Detection via filter.
+	/**
+	 * Filter to detect whether the current environment uses VirtualBox.
```

Comment:
```suggestion
	 * Filters whether the current environment uses VirtualBox.
```
Filter docblocks should begin with "Filters"